### DISCUSSION TITLE
Lesson 7 : test doesnt run

### DISCUSSION THREAD

**Discussion by karsar69** [Cyfrin/foundry-full-course-cu#165] [Q&A] [answered]
when running test   function ```ownerisMsgSender``` doesnt run ![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/119435383/38fd9dfb-aadd-45ad-be82-ecf4cd93d912)
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/119435383/c9b30d73-f006-47f1-a7e4-b06f6d4c9deb)

**Comment by allwin199** [comment, marked as answer]
https://book.getfoundry.sh/forge/writing-tests

test: Functions prefixed with test are run as a test case

testOwnerIsMsgSender() --> this should work

**Reply by cromewar** [reply]
@karsar69 does @worksofallwin answer helped you?

**Reply by karsar69** [reply]
no changing function name won’t make any difference

**Reply by pyromaus** [reply]
it should. i just took out the 'test' prefix and forge skipped that test. @worksofallwin's answer seems to be correct

**Reply by cromewar** [reply]
Remember you should type the correct names both on the script and on the execution @karsar69

**Reply by karsar69** [reply]
ok got it

**Reply by SuperHumano** [reply]
It worked for me, Thanks!
